stable/prometheus-mongodb-exporter/README.md

```diff
@@ -51,6 +51,7 @@ podAnnotations:
 | `resources` | Pod resource requests and limits | `{}` |
 | `env` | Extra environment variables passed to pod | `{}` |
 | `securityContext` | Security context for the pod | See values.yaml |
+| `serviceAccountName` | Service account name for the pod | `default` |
```

Comment:
Most charts don't attach outside created service accounts to a deployment. You should instead include the `ServiceAccount`  yaml in the chart using the best practices that maorfr mentioned.

Reply:
Thank you @steven-sheehy and @maorfr . I have added a commit to adhere to the best practices.